public class ExceptionExample{
  public static void main(String[] args){
       try{
             int num=10/0;
             System.out.println("Result: " +num);
}
  catch(ArithematicException){
     System.out.println("Error: Cannot divide by zero!");
}
     System.out.println("Program continues...");
}}
